src/views/ModelPlan/ReadOnly/Beneficiaries/index.tsx

```diff
@@ -85,72 +94,72 @@ const ReadOnlyBeneficiaries = ({ modelID, clearance }: ReadOnlyProps) => {
         </p>
       )}
 
-      <div className="margin-bottom-4 padding-bottom-2 border-bottom-1px border-base-light">
-        <ReadOnlySection
-          heading={t('beneficiariesQuestion')}
-          list
-          listItems={beneficiaries?.map(translateBeneficiariesType)}
-          listOtherItem={beneficiariesOther}
-          notes={beneficiariesNote}
-        />
-
-        <div className="desktop:display-flex flex-justify">
-          <div
-            className={
-              treatDualElligibleDifferent === TriStateAnswer.YES
-                ? 'desktop:width-card-lg'
-                : ''
-            }
-          >
-            <ReadOnlySection
-              heading={t('dualEligibility')}
-              copy={
+      <div
+        className={`${
+          isViewingFilteredView
+            ? ''
+            : 'margin-bottom-4 padding-bottom-2 border-bottom-1px border-base-light'
+        }`}
+      >
+        {checkGroupMap(
+          isViewingFilteredView,
+          filteredQuestions,
+          'beneficiaries',
+          <ReadOnlySection
+            heading={t('beneficiariesQuestion')}
+            list
+            listItems={beneficiaries?.map(translateBeneficiariesType)}
+            listOtherItem={beneficiariesOther}
+            notes={beneficiariesNote}
+          />
+        )}
+
+        {checkGroupMap(
+          isViewingFilteredView,
+          filteredQuestions,
+          'dualEligibility',
+          <SideBySideReadOnlySection
+            firstSection={{
+              heading: t('dualEligibility'),
+              copy:
                 treatDualElligibleDifferent &&
                 translateTriStateAnswer(treatDualElligibleDifferent)
+            }}
+            secondSection={
+              treatDualElligibleDifferent === TriStateAnswer.YES && {
+                heading: h('howSo'),
+                copy: treatDualElligibleDifferentHow
               }
-            />
-          </div>
-          {treatDualElligibleDifferent === TriStateAnswer.YES && (
-            <div className="desktop:width-card-lg">
-              <ReadOnlySection
-                heading={h('howSo')}
-                copy={treatDualElligibleDifferentHow}
-              />
-            </div>
-          )}
-        </div>
+            }
+          />
+        )}
         {treatDualElligibleDifferentNote && (
           <ReadOnlySection
             heading={t('basics:notes')}
             copy={treatDualElligibleDifferentNote}
           />
         )}
```

Comment:
This will render if a note exists, regardless if it's included in a group or not.  I think the notes should only show if their respective question is included in the group.  This shows in every view